boards/arty-e21/README.md

```diff
@@ -1,7 +1,3 @@
-SiFive E21 Core on ARTY FPGA Board
-=================
-
-- SiFive E21 Core IP v19p02
```

Comment:
Why remove this?

Reply:
This was an accident. Putting it back in.